src/index.js

```diff
@@ -429,6 +429,18 @@ f.analogWrite = function(pin, value, freq, callback) {
 
     // Make sure the pin has a PWM associated
     if(typeof pin.pwm == 'undefined') {
+        if(typeof pin.gpio != 'undefined' ) {
+            if(value>=0.5) {
+                resp=f.digitalWrite(pin,g.HIGH,callback);
+                if(callback) callback(resp);
+                return(true);
+            }
+            else {
```

Comment:
I think the 2 lines above probably go together to match the style. I'm going to go ahead and merge for now and work on beautifying the whole set of source code before enforcing any particular style.

Reply:
Thank you :)